state/application_ports.go

```diff
@@ -185,35 +184,114 @@ func (p *applicationPortRanges) removeOps() []txn.Op {
 	}}
 }
 
+type applicationPortRangesForUnit struct {
```

Comment:
I wonder if this could be called "podPortRangesForUnit"
Then we would have port ranges for machine (iaas) and pod (caas). ie where are the units grouped together.
"applicationPortRangesForUnit: and "applicationPortRangesForApplication" is confusing. Or if we really are saying that we are dealing with application ports, that's not until next cycle right?

Reply:
Actually, if we are just using this struct the record ports for the application (set via open-port --app) and unit (set via open-port) then the name of the struct could just be "portRages" or "applicationPortRanges", with methods to get either the --app ports or individual unit ports?

Reply:
As discussed, we rename the `applicationPortRangesForUnit` to `unitPortRanges`.